### ISSUE TITLE
no transform XSLT for Generic (don't worry about CDL source types)

### ISSUE THREAD

**Issue by drh-stanford** [sul-dlss/web-archiving#10]
See https://github.com/sul-dlss/was_robot_suite/tree/master/template/wasSeedPreassembly

The question here is whether this is correct. Currently, we're seeing errors in the 2nd step of the wasSeedPreassemblyWF when the metadata source is NOT ArchiveIT (AIT)

here's an example error message from the robot:

```
error: desc-metadata-generator : No such file or directory @ rb_sysopen - /home/lyberadmin/was_robot_suite/releases/20160628174800/template/wasSeedPreassembly/descMetadata_G.xslt
```

**Comment by drh-stanford** [comment]
Here are the values in the pull down for uploading seeds:

![screen shot 2016-06-27 at 4 02 04 pm](https://cloud.githubusercontent.com/assets/1861171/16398605/84138aa2-3c80-11e6-9fbf-fbe436b3c673.png)

**Comment by ndushay** [comment]
fixed by sul-dlss/was_robot_suite#11
